nuxt.config.ts

```diff
@@ -13,6 +13,11 @@ export default defineNuxtConfig({
 			},
 		],
 		link: [
+			{
+				rel: 'icon',
+				type: 'image/png',
+				href: 'btz.png'
+			},
```

Comment:
We definitely need a favicon but the one provided is difficult to see. 

![image](https://user-images.githubusercontent.com/294987/202043021-95d3f8cd-3037-4446-bc15-d0ba53ee179b.png)


The image size is also quite large. I usually use https://realfavicongenerator.net/ to generate the favicon.